packages/test/test-service-load/testConfig.json

```diff
@@ -12,16 +12,14 @@
             "faultInjectionMaxMs": 150000,
             "optionOverrides":{
                 "odsp":{
-                    "loader":{
-                        "summarizeProtocolTree": [false]
+                    "configurations":{
+                        "Fluid.Container.summarizeProtocolTree": [true, false]
                     }
                 },
                 "odsp-odsp-df":{
-                    "loader":{
-                        "summarizeProtocolTree": [false]
-                    },
                     "configurations":{
-                        "Fluid.Driver.Odsp.snapshotFormatFetchType": [0, 1]
+                        "Fluid.Driver.Odsp.snapshotFormatFetchType": [0, 1],
```

Comment:
Unrelated to your changes: Do we record these configs in telemetry in some form? It feels like it would be useful to differentiate runs that use different settings. Thoughts?

Reply:
not by default. we could consider logging on first read or something, as we don't allow the value to change during the lifetime of the session

Reply:
currently we expect the consuming code to log whatever details it finds necessary